dts/arm/st/g4/stm32g4.dtsi

```diff
@@ -302,6 +302,17 @@
 			label = "SPI_3";
 		};
 
+		spi4: spi@40013c00 {
```

Comment:
Please move to stm32g474.dtsi file as this node isn't available on g431 that includes this file.

Reply:
Makes sense. Done.